api/primary_test.go

```diff
@@ -56,8 +57,43 @@ func TestCorsRequest(t *testing.T) {
 		t.Log(k, " : ", v)
 	}
 
-	if w.Code == 404 {
-		t.Fatalf("failed not found")
+	// test response status code
+	if w.Code != 200 {
+		t.Fatalf("HTTP response status code is %d, not 200", w.Code)
+	}
+
+	// test Access-Control-Allow-Headers in response headers
+	// allowHeaders has the type of slice with only one string element
+	allowHeaders, exist := w.Header()["Access-Control-Allow-Headers"]
+	if !exist {
+		t.Fatal("no Access-Control-Allow-Headers in response")
+	}
+
+	if len(allowHeaders) != 1 {
```

Comment:
If it exists, how could it be anything but one? I don't mind checking, but I do want to understand.

0 could not happen, because it would not `exist` and we would have failed above.

Could 2 or more happen? I guess that would mean multiple headers with the same key value.

I'd like a minimal comment of `we return one and only one copy of this header`, I think.

Does that make sense?